Isabelle/Chi_Calculus_Examples/Diamond.thy

```diff
@@ -194,26 +194,13 @@ proof -
     sorry
   then have "a \<triangleright>\<^sup>\<infinity> x. \<Prod> c\<leftarrow>[b]. c \<triangleleft> x \<approx>\<^sub>\<flat> a \<triangleright>\<^sup>\<infinity> x. b \<triangleleft> x"
     by (simp add: basic_weak_multi_receive_preservation)
-  then show ?thesis .
+  then show ?thesis
+    by simp
```

Comment:
Why did you replace `.` with `by simp`?

Reply:
Because suddenly `.` stopped working, I guess it has to do with the new definitions.

Reply:
Well, this particular lemma should be removed anyhow. 🙂

Reply:
That said, it’s clear why `.` doesn’t work here anymore: the right-hand side of the chained fact is the old expansion of `a → b`, not the new one.